packages/application/style/titlepanel.css

```diff
@@ -0,0 +1,29 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+:root {
+  --jp-private-title-panel-height: 28px;
+}
+
+#jp-title-panel {
+  min-height: var(--jp-private-title-panel-height);
+  width: 100%;
+  display: flex;
+  background: var(--jp-layout-color1);
+  overflow: visible;
+}
+
+#jp-TitlePanel-title {
```

Comment:
Since this is an ID, should it be `#jjp-title-panel-title`?

Reply:
Hm, it appears we are inconsistent, cf. `#jp-MainLogo`.

Reply:
Fixed in latest commit.